src/aics-image-viewer/components/App/index.tsx

```diff
@@ -322,6 +318,7 @@ const App: React.FC<AppProps> = (props) => {
     if (aimg.isLoaded()) {
       view3d.updateActiveChannels(aimg);
       setImageLoaded(true);
+      initialLoadRef.current = false;
```

Comment:
it's surprising to me there isn't a setRef? instead of manually setting the `.current`

Reply:
The idea is that refs aren't values from which the rendered component is derived, so you don't need to let React know that you're updating them and that it should potentially re-render. I think their primary/original purpose is to attach to rendered DOM elements so that you can do imperative stuff to them outside the React cycle, though they're also legal to use in cases like this one, where the value influences some side effect of a future render without changing its result.